data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -255,8 +187,7 @@ public Object process(org.apache.hadoop.hive.ql.lib.Node nd, Stack<org.apache.ha
 	}
 
 	private void getTableLineage(ASTNode queryNode) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "getTableLineage" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)